src/main.cpp

```diff
@@ -33,8 +36,38 @@ void loop()
 	for (int j = 0; j < 255; j++)
 	{
 		for (int i = 0; i < NUM_LEDS; i++)
-			leds[i] = CHSV((i+j)%254, 254, 254);
+			leds[i] = CHSV((i+j)%256, 255, 64);
```

Comment:
Would be good if you used a variable for controlling brightness. Could even be a #define up the top if you're lazy 👍

Reply:
Updating with an int because I'm not a scrub